integration/spark-common/src/main/scala/org/apache/carbondata/spark/CarbonOption.scala

```diff
@@ -23,51 +23,59 @@ package org.apache.carbondata.spark
  */
 class CarbonOption(options: Map[String, String]) {
 
-  def dbName: Option[String] = options.get("dbName")
+  lazy val dbName: Option[String] = options.get("dbName")
 
-  def tableName: String = options.getOrElse("tableName", "default_table")
+  lazy val tableName: String = options.getOrElse("tableName", "default_table")
 
-  def tablePath: Option[String] = options.get("tablePath")
+  lazy val tablePath: Option[String] = options.get("tablePath")
 
-  def partitionCount: String = options.getOrElse("partitionCount", "1")
+  lazy val partitionCount: String = options.getOrElse("partitionCount", "1")
 
-  def partitionClass: String = {
+  lazy val partitionClass: String = {
     options.getOrElse("partitionClass",
       "org.apache.carbondata.processing.partition.impl.SampleDataPartitionerImpl")
   }
 
-  def tempCSV: Boolean = options.getOrElse("tempCSV", "false").toBoolean
+  lazy val tempCSV: Boolean = options.getOrElse("tempCSV", "false").toBoolean
```

Comment:
I remember that the 'tempCsv' option has been deprecated

Reply:
removed